### DISCUSSION TITLE
More speed for folding or unfolding by clicking with the mouse inside the node

### DISCUSSION THREAD

**Discussion by aaa1386** [freeplane/freeplane#1456] [Help] [answered]
Clicking the mouse inside the node and folding or unfolding is very interesting and practical, but it works slowly
Does the software have the ability to increase the speed of this operation?

**Comment by sifran-github** [comment]
I can tell you what I do, maybe it will help: I disable this feature completely. I never want to fold/unfold just by clicking a node.
I use the space keyboard shortcut, so it is very fast to mark a node and press space. and I also use the oval fold/unfold next to nodes.

**Comment by aaa1386** [comment]
Sometimes the mouse is in your hand. It is easier than the shortcut
Of course, most of the time the shortcut is easier, but not always

**Comment by dpolivaev** [comment, marked as answer]
Disable Preferences…->Behaviour->Edit on double click

**Comment by dpolivaev** [comment]
By the way, if you @ajaberi1360 want to ask a question like in this case, please use discussions and not issues.

**Reply by aaa1386** [reply]
ok sure
